app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,27 @@
+class EntitiesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @group = Group.includes(:entities).find_by_id(params[:group_id])
+    @entities = @group.entities.order(created_at: :desc)
+  end
+
+  def new
+    @group = Group.find(params[:group_id])
+    @entity = @group.entities.new
+  end
+
+  def create
+    @group = Group.find(params[:group_id])
+    @entity = @group.entities.new({ author_id: current_user.id, **entity_params })
+    @entity.save
+
+    redirect_to group_entities_path(@group)
+  end
```

Comment:
- [ ] It is always a great practice to create success and failure paths. What if the entity is not created for some reason? You should also handle that scenario.